packages/patternfly-4/react-core/src/components/Pagination/Navigation.tsx

```diff
@@ -33,8 +33,10 @@ export interface NavigationProps extends React.HTMLProps<HTMLElement> {
   paginationTitle?: string;
   /** The number of the current page */
   page: React.ReactText;
+  /** Number of items per page. */
+  perPage?: number;
   /** Function called when page is changed */
-  onSetPage: (event: React.SyntheticEvent<HTMLButtonElement>, page: number) => void;
+  onSetPage: (_evt: React.MouseEvent | React.KeyboardEvent | MouseEvent, newPage: number, perPage?: number, startIdx?: number, endIdx?: number) => void;
```

Comment:
Can we define and share a type alias for this, since it's used in multiple places?